hyundai_kia_connect_api/KiaUvoApiEU.py

```diff
@@ -216,18 +217,18 @@ def update_vehicle_with_cached_state(self, token: Token, vehicle: Vehicle) -> No
         vehicle.side_mirror_heater_is_on = get_child_value(
             state, "vehicleStatus.sideMirrorHeat"
         )
-        vehicle.front_left_seat_heater_is_on = get_child_value(
+        vehicle.front_left_seat_status = SEAT_STATUS[get_child_value(
```

Comment:
overall, should we stick with `front_left_seat_heater_status` and `SEAT_HEAT_STATUS`? or do we store any state other than heat here?

Reply:
We store states including cool.   The constant file has the text states for your reference.

Reply:
https://github.com/Hyundai-Kia-Connect/hyundai_kia_connect_api/blob/783b6abace0fc9154f9ea4c93f4aabb43dbaf440/hyundai_kia_connect_api/const.py#L22